### PR TITLE
Remove namespaced thrift step2

### PR DESCRIPTION
**Pull request by raronson**
This is the follow on from #678. Changes:

  * Output format of chord and squash are partitioned by namespace, and stored with the date as key and `ThriftFact` as value.
  * `GroupByEntityOutputJob` uses `ThriftFact` instead of `NamespacedThriftFact` when passing facts from mappers to reducers.
  * New partitioned emitter because squash reducer outputs to multiple namespaces.

### CODE REVIEW

File: ivory-core/src/main/scala/com/ambiata/ivory/core/Fact.scala [hunk 1]
```diff
@@ -89,53 +89,6 @@ object Fact {
...
-
-    def value: Value =
-      Value.fromThrift(fact.getValue)
-
-    def toThrift: ThriftFact = fact
-
-    def toHydrated: HydratedFact =
-      HydratedFact(namespace, date, toThrift)
-
-    def toNamespacedThrift = this
```
Comment (charleso) [inline ivory-core/src/main/scala/com/ambiata/ivory/core/Fact.scala]: You have a `TODO` in the other PR for removing this from the `trait`. It looks like the only place left is in `NamespacedFactWriter`, can/should that be removed?

File: ivory-storage/src/main/scala/com/ambiata/ivory/storage/fact/Namespaces.scala [hunk 2]
```diff
@@ -37,10 +37,16 @@ object Namespaces {
...
-  def fromSnapshotPath(path: Path): Namespace = {
+  def fromPathUnsafe(path: Path): Namespace = {
     Namespace.nameFromStringDisjunction(path.getParent.getName) match {
       case scalaz.\/-(n) => n
-      case scalaz.-\/(e) => Crash.error(Crash.DataIntegrity, s"Can not parse snapshot namespace from path '${path}' - ${e}")
+      case scalaz.-\/(e) => Crash.error(Crash.DataIntegrity, s"Can not parse namespace from path '${path}' - ${e}")
     }
   }
+
+  def sizeNamespaces(root: Path): Hdfs[List[Sized[Namespace]]] =
```
Comment (charleso) [inline ivory-storage/src/main/scala/com/ambiata/ivory/storage/fact/Namespaces.scala]: Worth adding a separate test for?

**Comment by charleso** [comment]
:baby: 

FYI the cli tests are failing, although looks like a map ordering problem

File: ivory-core/src/main/scala/com/ambiata/ivory/core/Fact.scala (hunk 1 above)
Comment (raronson) [inline ivory-core/src/main/scala/com/ambiata/ivory/core/Fact.scala]: thanks, i missed this, removing

File: ivory-storage/src/main/scala/com/ambiata/ivory/storage/fact/Namespaces.scala (hunk 2 above)
Comment (raronson) [inline ivory-storage/src/main/scala/com/ambiata/ivory/storage/fact/Namespaces.scala]: actually, i think its a duplicate of `namespaceSizes`, so will remove

**Comment by nhibberd** [comment]
@raronson status

**Pull request merged**